<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMessageBox</class>
 <widget class="QDialog" name="CMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="widgetGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{background:white;}</string>
     </property>
     <layout class="QGridLayout" name="mainWidgetGridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="titleBarWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <layout class="QGridLayout" name="titleBarWidgetGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="titleBarLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="iconLabel">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="titleLabel">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Socket</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="titleBarLayoutHSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="minimizeButton">
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>最小化</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#e2e2e2;}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../style/style.qrc">
               <normaloff>:/custom/icons/minimize.png</normaloff>:/custom/icons/minimize.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="closeButton">
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>关闭</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#fb7373;}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../style/style.qrc">
               <normaloff>:/custom/icons/close.png</normaloff>:/custom/icons/close.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="infoWidget" native="true">
        <layout class="QGridLayout" name="infoWidgetGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="infoLabel">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="bottomWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QGridLayout" name="bottomWidgetGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="buttonsLayout">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QPushButton" name="confirmButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>10</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{border-radius:10px;font:10pt &quot;Microsoft YaHei UI&quot; ;color:#ffffff;background:#07c160}
QPushButton:hover{background-color: rgb(56, 205, 127);}</string>
               </property>
               <property name="text">
                <string>确认</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="copyButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>9</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{border-radius:10px;font:9pt &quot;Microsoft YaHei UI&quot; ;color:#ffffff;background:#07c160}
QPushButton:hover{background-color: rgb(56, 205, 127);}</string>
               </property>
               <property name="text">
                <string>复制信息</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancelButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>10</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{border-radius:10px;font:10pt &quot;Microsoft YaHei UI&quot; ;color:#ffffff;background:#fb7373}
QPushButton:hover{background-color: rgba(251, 115, 115,80%);}</string>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../style/style.qrc"/>
 </resources>
 <connections/>
</ui>
